spec/schemata/queryable_spec.exs

```diff
@@ -236,12 +236,22 @@ defmodule Schemata.Queryable.AlterTableSpec do
     {:shared, queries: [
       {"ALTER TABLE users ALTER email TYPE text",
        %AlterTable{named: "users", column: :email, op: {:alter, :text}}},
+      {"ALTER TABLE users ALTER contacts TYPE list<text>",
+       %AlterTable{named: "users", column: :contacts,
+                   op: {:alter, {:list, :text}}}},
       {"ALTER TABLE users ADD email text",
        %AlterTable{named: "users", column: :email, op: {:add, :text}}},
+      {"ALTER TABLE users ADD friends set<text>",
+       %AlterTable{named: "users", column: :friends,
+                   op: {:add, {:set, :text}}}},
       {"ALTER TABLE users RENAME email TO mail",
        %AlterTable{named: "users", column: :email, op: {:rename, :mail}}},
       {"ALTER TABLE users DROP email",
        %AlterTable{named: "users", column: :email, op: :drop}},
+      {"ALTER TABLE users DROP email",
+       %AlterTable{named: "users", column: :email, op: :drop}},
+      {"ALTER TABLE users DROP email",
+       %AlterTable{named: "users", column: :email, op: :drop}},
```

Comment:
That's it - I'm banned from using copy-paste until I learn to clean up after myself.